tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -562,6 +565,7 @@ describe(__filename, () => {
 
     sinon.assert.calledWith(dispatchSpy, editUserAccount({
       errorHandlerId: errorHandler.id,
+      notifications: {},
```

Comment:
There doesn't seem to be a test that covers sending a new notifications object to `editUserAccount`. It could maybe be added to this test case, or perhaps you want to add a new test.

Reply:
ah right, thank you. We test that notifications get passed and we test `editUserAccount` on its own though, but one more test won't hurt :+1: